include/CppUTest/Shuffle.h

```diff
@@ -44,15 +44,18 @@ static inline int rand_(void) // rand_func_t
 }
 
 // "Durstenfeld shuffle" according to Wikipedia
-static inline void shuffle_list(rand_func_t rand_func, int numElems, void* listToShuffleInPlace[])
+static inline void shuffle_list(rand_func_t rand_func, size_t numElems, void* listToShuffleInPlace[])
 {
-    for (int i = numElems - 1; i >= 1; --i)
+    if( numElems > 0 )
```

Comment:
Could you change this to:

if (numElems == 0) return;

So that the code is easier to read and not indented